### PR TITLE
Migrate name to labels, adding context-object

### PR DESCRIPTION
**Pull request by brinxmat**
- name is labels
- add context object

### CODE REVIEW

**Review by torarnet — COMMENTED** [review]
Update karate tests with new field name and validate context object is displayed correctly

File: cristin-funding/src/main/java/no/unit/nva/cristin/funding/sources/model/nva/FundingSource.java [hunk 1]
```diff
@@ -21,15 +31,15 @@ public class FundingSource {
     @JsonProperty(IDENTIFIER_FIELD)
     private final String identifier;
     @JsonProperty(NAME_FIELD)
```
Comment (torarnet) [inline cristin-funding/src/main/java/no/unit/nva/cristin/funding/sources/model/nva/FundingSource.java:33]: you forgot to update name tag

**Review by torarnet — CHANGES_REQUESTED** [review]
You forgot to update field's name tag. Also, you need to update karate tests

**Comment by ketilaa** [comment]
@brinxmat Did you forget about this PR? Is it something we should work to complete?

**Comment by torarnet** [comment]
status @brinxmat ?

**Pull request merged**